modules/ROOT/nav.adoc

```diff
@@ -9,6 +9,7 @@
  ** xref:prereq-verify-device.adoc[Verify Device Requirements]
  ** xref:prereq-network.adoc[Network Requirements]
  ** xref:prereq-software.adoc[About Software Requirements]
+ ** xref:prereq-env.adoc[Environment Limitations]
```

Comment:
Should this be included here or in the release notes? I'm not sure about the pros and cons.

Reply:
I think it should be included here, unless it's something that's going to be fixed. Putting this in the release notes would mean classifying it as a 'Known Issue', which implies that it should be fixed. This information looks like a long term situation, which means it should go in the docs. Thoughts?